tests/NexyCryptTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Nexy\NexyCrypt\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Nexy\NexyCrypt\NexyCrypt;
+use Nexy\NexyCrypt\Exception\AcmeApiException;
+
+class NexyCryptTest extends TestCase
+{
+    public $url = 'https://acme-staging.api.letsencrypt.org/';
+
+    public $domain = 'nexycrypt.esy.es';
+
+    public $keyPath = '/nexycrypt.private_key';
+
+    /** @test */
```

Comment:
Yes, but I don't think this is necessary. All method with `Test` suffix are called by default.